Modules/UserRoles.cs

```diff
@@ -74,8 +74,6 @@ public static async Task RemoveUserRolesAsync(CommandContext ctx, Func<System.Re
                 DiscordRole roleToGrant = guild.GetRole((ulong)roleId.GetValue(Program.cfgjson.UserRoles, null));
                 await ctx.Member.RevokeRoleAsync(roleToGrant);
             }
-
-            await ctx.Message.DeleteAsync();
```

Comment:
Attempting to delete a message multiple times causes an exception

Reply:
Maybe you could try/catch it? Failing to delete the message isn't an exception worth acting on after all, can just be discarded.